spec/unit/helper_builder_spec.rb

```diff
@@ -284,15 +285,55 @@
     end
   end
 
+  describe "#buildable_archs" do
+    before(:each) do
+      allow(subject).to receive(:run_uname_p).and_return("x86_64")
+    end
+
+    context "for Go 1.4.2" do
+      before(:each) do
+        allow(subject).to receive(:run_go_version).and_return("go version go1.4.2 linux/amd64")
+      end
+
+      it "returns only the local arch if it is x86_64" do
+        expect(subject).to receive(:run_uname_p).and_return("x86_64")
```

Comment:
You are right but it is easier to understand I guess but I don't mind removing it.